src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -68,19 +61,43 @@ export default function VAFacilityPageV2() {
     schema,
     selectedFacility,
     showEligibilityModal,
+    showVariant,
     singleValidVALocation,
     sortMethod,
     typeOfCare,
   } = useSelector(state => getFacilityPageV2Info(state), shallowEqual);
+  const sortOptions = [
+    { value: 'distanceFromResidentialAddress', label: 'By your home address' },
+    { value: 'distanceFromCurrentLocation', label: 'By your current location' },
+    { value: 'alphabetical', label: 'Alphabetically' },
+  ];
+  const [sortType, setSortType] = useState(sortMethod);
+
+  const uiSchema = {
+    vaFacility: {
+      'ui:title': showVariant
+        ? `Select a VA facility where you’re registered that offers ${lowerCase(
+            typeOfCare?.name,
+          )} appointments.`
+        : 'Please select where you’d like to have your appointment.',
+      'ui:widget': FacilitiesRadioWidget,
+    },
+  };
+
   const loadingEligibility = loadingEligibilityStatus === FETCH_STATUS.loading;
   const loadingFacilities =
     childFacilitiesStatus === FETCH_STATUS.loading ||
     childFacilitiesStatus === FETCH_STATUS.notStarted;
-  const pageTitle = singleValidVALocation
-    ? 'Your appointment location'
-    : `Choose a VA location for your ${lowerCase(
-        typeOfCare?.name,
-      )} appointment`;
+  let pageTitle;
+  if (singleValidVALocation) {
+    pageTitle = 'Your appointment location';
+  } else if (showVariant) {
+    pageTitle = 'Choose a VA Location';
+  } else {
+    pageTitle = `Choose a VA location for your ${lowerCase(
```

Comment:
Unrelated to copy updates for the variant test, but it looks like this should also have been lowercased.